$NetBSD: patch-Gemfile,v 1.4 2022/12/03 06:41:42 taca Exp $

* Relax dependency.
* Do not load gem for developemt and test.

--- Gemfile.orig	2022-12-01 15:40:32.000000000 +0000
+++ Gemfile
@@ -3,9 +3,9 @@ source 'https://rubygems.org'
 ruby '>= 2.5.0', '< 3.2.0'
 gem 'bundler', '>= 1.12.0'
 
-gem 'rails', '6.1.7'
+gem 'rails', '~>6.1'
 gem 'globalid', '~> 0.4.2' if Gem.ruby_version < Gem::Version.new('2.6.0')
-gem 'rouge', '~> 3.28.0'
+gem 'rouge', '~> 3.28'
 gem 'request_store', '~> 1.5.0'
 gem 'mini_mime', '~> 1.1.0'
 gem "actionpack-xml_parser"
@@ -48,7 +48,7 @@ gem 'redcarpet', '~> 3.5.1', groups: [:m
 
 # Optional CommonMark support, not for JRuby
 group :common_mark do
-  gem "html-pipeline", "~> 2.13.2"
+  gem "html-pipeline", "~> 2.13"
   gem "commonmarker", (Gem.ruby_version < Gem::Version.new('2.6.0') ? '0.21.0' : '~> 0.23.6')
   gem "sanitize", "~> 6.0"
   gem 'deckar01-task_list', '2.3.2'
@@ -69,7 +69,7 @@ if File.exist?(database_file)
       when 'mysql2'
         gem "mysql2", "~> 0.5.0", :platforms => [:mri, :mingw, :x64_mingw]
       when /postgresql/
-        gem "pg", "~> 1.2.2", :platforms => [:mri, :mingw, :x64_mingw]
+        gem "pg", "~> 1.2", :platforms => [:mri, :mingw, :x64_mingw]
       when /sqlite3/
         gem "sqlite3", "~> 1.4.0", :platforms => [:mri, :mingw, :x64_mingw]
       when /sqlserver/
@@ -86,6 +86,7 @@ else
   warn("Please configure your config/database.yml first")
 end
 
+if false
 group :development do
   gem 'listen', '~> 3.3'
   gem "yard"
@@ -107,6 +108,7 @@ group :test do
   gem 'rubocop-performance', '~> 1.13.0'
   gem 'rubocop-rails', '~> 2.14.0'
 end
+end
 
 local_gemfile = File.join(File.dirname(__FILE__), "Gemfile.local")
 if File.exist?(local_gemfile)
